src/Nethermind/Nethermind.Consensus.Ethash/EthashDifficultyCalculator.cs

```diff
@@ -10,6 +10,7 @@
 
 [assembly: InternalsVisibleTo("Ethereum.Test.Base")]
 [assembly: InternalsVisibleTo("Ethereum.Difficulty.Test")]
+[assembly: InternalsVisibleTo("Evm")]
```

Comment:
why was this added here?

Reply:
Used to calculate difficulty in ApplyMergeChecks as per logic of geth